ansible/roles/flannel/files/40-docker-depends-on-flannel.conf

```diff
@@ -0,0 +1,9 @@
+# Dropin for docker.service
+
+[Unit]
+# Flannel must be started before Docker.
```

Comment:
@rutsky do you mind adding a link to the coreos url you mentioned in your feedback as a comment? I think this may be it: https://github.com/coreos/flannel/issues/112